### DISCUSSION TITLE
Customer key with Istio (HTTPS to HTTP)

### DISCUSSION THREAD

**Discussion by Cl0udius** [minio/minio#14905] [Q&A]
Hi,

we are currently working on a data exchange solution between two service which should work with SSE based on a customer key.
Our setup also includes Istio sidecar which adds a proxy container to the pod.
The proxy container will in our case terminate the TLS connection and forward the request internally over pure HTTP.

When we now try to upload data with our custom key we receive the following message:

ERROR:root:An error occurred (InvalidRequest) when calling the PutObject operation: Requests specifying Server Side Encryption with Customer provided keys must be made over a secure connection.

It seems that due to the missing https connection between istio-proxy and the minio pod we run into this error.

So is there a way on Minio side to turn off this behavior by a setting in the Helm chart?

BR
Alex

**Comment by 4n70w4** [comment]
Similar issue: https://github.com/minio/minio/issues/6093
